src/components/CodeAssignmentModal/CodeAssignmentModal.test.jsx

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import { screen, render } from '@testing-library/react';
+// import userEvent from '@testing-library/user-event';
```

Comment:
should we just remove this line then?

Reply:
Nooooo, I left it in because of the commented-out test that I kind of wanted ideas on whether to either
1) skip this test entirely, even though it's a useful test
2) change this testing file to the way I did the other testing file, by mocking `Field`, which makes some of the current tests difficult
3) figure out how to get jest to mock something for JUST ONE TEST. Like, it should be straightforward, but I tried a bunch of things and nothing work
4) Move this test to a different file so that I can do the mock and run the test. I hate it in theory but it would get the job done.

Reply:
I know there is a `mockReturnValueOnce` for functions. Looking at the test, I feel like I had this same issue with submission as well where it just didn't do the thing properly, so I don't know that I can be helpful in determining a solution. :/